### PR TITLE
[T4A5][W15-B4]Tang Yew Siang

### PR DESCRIPTION
**Pull request by yewsiang**
Ready for review.

### CODE REVIEW

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.tag;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:2]: Please include a header comment to indicate what this interface/class/method does?

File: src/seedu/addressbook/data/tag/TaggingList.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/TaggingList.java:4]: Please include a header comment to indicate what this interface/class/method does?

**Comment by okkhoy** [comment]
@yewsiang pretty good work on this PR. keep it up :+1: 
some minor comments added, please take a look; ack and close the PR.

**Comment by yewsiang** [comment]
Thank you :)!

**Pull request closed without merging**